pkg/dao/snapshots.go

```diff
@@ -224,6 +224,7 @@ func (sDao *snapshotDaoImpl) FetchForRepoConfigUUID(ctx context.Context, repoCon
 	var snaps []models.Snapshot
 	result := sDao.db.WithContext(ctx).Model(&models.Snapshot{}).
 		Where("repository_configuration_uuid = ?", repoConfigUUID).
+		Order("created_at desc").
```

Comment:
there's nothing inherent about this function that would mean it has to be ordered by created_at, but the task calling the function depends on it.  I might would either simply add a comment explaining its purpose, rename the function to indicate that, or create a new function FetchLatestSnapshot()  that just returns the latest.  
I could see someone changing it in the future to suit some other need without realizing why its returning in this order

Reply:
oh actually there is one! FetchLatestSnapshot()

Reply:
I agree, updated